### ISSUE TITLE
NOCOMP model with accelerated decomposition 

### ISSUE THREAD

**Issue by rosiealice** [NorESMhub/NorESM3_land_only_dev_simulations#8]
**Purpose:**

___
**Description:**

___
**Compset long name**
1850_DATM%QIA_CLM60%FATES-SP_SICE_SOCN_MOSART_SGLC_SWAV
___
**Model grid long name**
ne30pg3_tn14
___
**Case directory:**
- Locally on betzy (if still available):
`/projects/NS9560K/noresm3/cases/$user/$CASE`

- On github:
https://github.com/NorESMhub/noresm_simulations/tree/$CASE
___
**Code version on github (either NorESMhub or user's fork):**
- https://github.com/NorESMhub/noresm/tree/<code_version>
- https://github.com/username/noresm/tree/<code_version>
___
**Diagnostics:**
- Diags (if available):
As examples:
- https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/<$CASE_vs_$BASELINE>/website/index.html
- https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/<$CASE_vs_$OBS>/website/index.html

___
**Output:**
- Raw output on nird (if still available):
- CAM Climos:
___
**Contacts:**

___
**Extra details:**

**Comment by kjetilaas** [comment]
I tested this, and got the following error message:


Using cold start initial conditions 
 hist_htapes_build Initializing clm2 history files
------------------------------------------------------------
 htapes_fieldlist ERROR: GPP in fincl(           4 ) for history tape 
           1  not found
 ENDRUN:
 ERROR: ERROR in histFileMod.F90 at line 887

**Comment by kjetilaas** [comment]
Here is the link to the setup (using commit df05388):
https://github.com/kjetilaas/Run_NorESM_script

Only difference from description above is that compset is SROF instead of MOSART. 

Link to case on betzy (temporary):  
/cluster/work/users/kjetisaa/noresm/n1850.FATES-NOCOMP-AD.ne30_tn14.alpha08d.20250102/run

The same setup (script) works without accelerated decomp.
And the same setup (script) works with CLM (not FATES) and AD.

**Comment by rosiealice** [comment]
Looks like it is the GPP variable (which is a CLM variable) in the fincl which is causing the issue. I wonder if this is turned on by default when we turn on AD? Is it present in tbe user_nl_clm ?

**Comment by kjetilaas** [comment]
user_nl_clm only includes:

fates_paramfile = '/cluster/work/users/rosief/git/calibration_parameter_files/SP_calib/fates_params_det_alpha09.nc'
use_fates_nocomp=.true.
use_fates_fixed_biogeog=.true.

**Comment by rosiealice** [comment]
Right. So it must be being set automatically... Is there an fincl in the lnd_in file in the run directory?

**Comment by kjetilaas** [comment]
Yes, it is. Adding this line to the user_nl_clm solved the problem:
hist_fincl1='TOTSOMC','TOTSOMN','TLAI','FATES_GPP','FATES_NPP','TWS'

**Comment by rosiealice** [comment]
Oh cool. That's good news...
